client/debian/changelog

```diff
@@ -1,3 +1,9 @@
+rust-hwlib (0.9.2) questing; urgency=medium
+
+  * Replace lsb-release dependency with os-release Rust library
+
+ -- Pedro Avalos <pedro.avalosjimenez@canonical.com>  Sat, 17 May 2025 09:31:10 +0200
+
```

Comment:
We shouldn't create a new release with this change, this should be handled later when we create a new release (moreover, 0.9.1 has not been published, we can do it with 0.9.1)

Reply:
Just add this point under the 0.9.1 release